tests/test_oracle_grounded_record.py

```
@@ -0,0 +1,733 @@
+from pathlib import Path
+
+REPO = Path(__file__).resolve().parents[1]
+sys.path.insert(0, str(REPO / "pipelines"))
+
+from oracle_grounded import canon, families, generators, oracles, record  # noqa: E402
+
+DOUBLE = REPO / "tests" / "fixtures" / "oracle-grounded" / "protocol_double.py"
+PINNED_COMMIT = "0" * 40
+
+
+def build(family, index=0, **kwargs):
```

Ruff (0.16.2)

[warning] Missing type annotation for `**kwargs`

(ANN003)